tests/ballerina-unit-test/src/test/resources/test-src/main.function/test_basic_main_function.bal

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Incorrect year or file was renamed? Better to update the year of these files as well.